src/emitters/emitter.sqns.ts

```diff
@@ -628,6 +629,15 @@ export class SqnsEmitter implements IEmitter {
     queue.consumer.start();
```

Comment:
queue.consumer's type should be changed to array of Consumer otherwise we will loose reference to the created consumer. Changing the type will expose any other places that need modification. Should not be a lot though